packages/widgets/src/docklayout.ts

```diff
@@ -2184,7 +2193,7 @@ namespace Private {
     renderer: DockLayout.IRenderer
   ): TabLayoutNode {
     // Create the tab bar for the layout node.
-    let tabBar = renderer.createTabBar();
+    let tabBar = renderer.createTabBar(this._document);
```

Comment:
Does this exist in this context?
This function is inside the Private namespace.

Reply:
I'm updating lumino in JupyterLab, and there is an issue with `this._document`, when changing to a single-mode document.
https://github.com/jupyterlab/jupyterlab/pull/11823

Reply:
Do you have an example of the breaking code - I couldn't see anything related in the failing visual regression test?

Reply:
Fix incomming.

Reply:
FYI:  https://github.com/jupyterlab/lumino/pull/281

Reply:
Hey @GordonSmith, thanks for the quick reply.

> Do you have an example of the breaking code

Yes, when in JupyterLab you change the view to a single-mode document, there is an error on this line:
https://github.com/jupyterlab/jupyterlab/blob/3b21d568035f7383dfd88332e051277130aea7d8/packages/application/src/shell.ts#L557

Following the trace, you can see that the error comes from
https://github.com/jupyterlab/lumino/blob/948eec9353d06ff9808dc127c888a49a9930ca75/packages/widgets/src/docklayout.ts#L2196

where `this` is null.